resource_management_mobile/src/app/auth/login.service.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, map } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import { LoginRequest, LoginResponse } from './models/auth.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+  URL = environment.baseUrl
```

Comment:
Can you add type here? 

private URL = ?

Reply:
Also, Format the code!

Reply:
Added Type and code formatted.